pages/common/hping.md

```diff
@@ -0,0 +1,25 @@
+# hping
+
+> Command-line oriented TCP/IP packet assembler and analyzer.
+> Inspired by the `ping` command.
+> More information: <http://www.hping.org>.
+
+- Ping localhost over TCP:
+
+`hping3 {{localhost}}`
+
+- Ping an IP address over TCP on a specific port:
+
+`hping3 -p {{80}} -S {{12.0.0.1}}`
```

Comment:
Let's change all the IPs to 192.168.1.1 so that they are something different and consistent with other pages.